zipline/gens/brokers/ib_broker.py

```diff
@@ -435,27 +445,32 @@ def account(self):
 
         z_account = zp.Account()
 
-        z_account.settled_cash = None  # TODO(tibor)
+        z_account.settled_cash = float(ib_account['TotalCashValue-S'])
         z_account.accrued_interest = None  # TODO(tibor)
         z_account.buying_power = float(ib_account['BuyingPower'])
         z_account.equity_with_loan = float(ib_account['EquityWithLoanValue'])
-        z_account.total_positions_value = None  # TODO(tibor)
-        z_account.total_positions_exposure = None  # TODO(tibor)
+        z_account.total_positions_value = float(ib_account['StockMarketValue'])
+        z_account.total_positions_exposure = float(
+            (z_account.total_positions_value /
+             (z_account.total_positions_value +
+              float(ib_account['TotalCashValue']))))
         z_account.regt_equity = float(ib_account['RegTEquity'])
         z_account.regt_margin = float(ib_account['RegTMargin'])
         z_account.initial_margin_requirement = float(
             ib_account['FullInitMarginReq'])
         z_account.maintenance_margin_requirement = float(
             ib_account['FullMaintMarginReq'])
-        z_account.available_funds = None  # TODO(tibor)
+        z_account.available_funds = float(ib_account['AvailableFunds'])
         z_account.excess_liquidity = float(ib_account['ExcessLiquidity'])
         z_account.cushion = float(
             self._tws.accounts[self.account_id]['']['Cushion'])
         z_account.day_trades_remaining = float(
             self._tws.accounts[self.account_id]['']['DayTradesRemaining'])
         z_account.leverage = float(
             self._tws.accounts[self.account_id]['']['Leverage-S'])
-        z_account.net_leverage = None  # TODO(tibor)
+        z_account.net_leverage = float(
```

Comment:
I think I found a better way to get the leverage:
`tws.accounts[account_id][None]['Leverage-S']`
{code}
            {'DU15141': defaultdict(<function zipline.gens.brokers.ib_broker.<lambda>>,
                         {None: defaultdict(<function zipline.gens.brokers.ib_broker.<lambda>>,
                                      {'AccountCode': 'DU15141',
                                       'AccountReady': 'true',
                                       'AccountType': 'UNIVERSAL',
                                       'Cushion': '0.501258',
                                       'DayTradesRemaining': '-1',
                                       'DayTradesRemainingT+1': '-1',
                                       'DayTradesRemainingT+2': '-1',
                                       'DayTradesRemainingT+3': '-1',
                                       'DayTradesRemainingT+4': '-1',
                                       'Leverage-S': '1.99',
                                       'LookAheadNextChange': '0',
                                       'SegmentTitle-C': 'Commodities',
                                       'SegmentTitle-S': 'Securities',
                                       'TradingType-S': 'STKNOPT',
                                       'WhatIfPMEnabled': 'true'}),
{code}

Do you think it makes sense to use this reported number?

I'm not sure why we have this under None key though..

Reply:
Leverage-S is ok as normal Leverage but if we want Net_leverage then it is another number. It sould be long - short.

When i tried that on Quantopian then
net_lev_calc = context.portfolio.positions_value/(context.portfolio.cash+context.portfolio.positions_value)
is same a
context.account.net_leverage

Reply:
Thanks for double checking the calculation, Jan! 
It is better to stay with the current calculation as it matches with Q's original platform.